sherpa/optmethods/__init__.py

```diff
@@ -17,6 +18,86 @@
 #  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 #
 
+"""Optimization classes.
+
+The `OptMethod` class provides an interface to a number of optimisers.
+When creating an optimizer an optional name can be added; this name is
+only used in string representations of the class:
+
+>>> opt = NelderMead()
```

Comment:
Add import statement from NelderMead, so I can see that it comes from this module.